src/java.base/share/classes/java/text/PatternEntry.java

```diff
@@ -52,14 +52,14 @@ class PatternEntry {
     /**
      * Gets the current extension, quoted
      */
-    public void appendQuotedExtension(StringBuffer toAddTo) {
+    public void appendQuotedExtension(StringBuilder toAddTo) {
```

Comment:
Hello Andrey, this and the other public method that's changed in this (package private) class don't seem to be used anywhere else other than this class itself. So maybe you could change them to `private` as part of this change?

Reply:
Changed a few of methods to `private`

Reply:
Thank you Andrey for doing those changes.